pom.xml

```diff
@@ -85,7 +85,7 @@
     <dependency>
       <groupId>com.ning</groupId>
       <artifactId>async-http-client</artifactId>
-      <version>1.7.24</version>
+      <version>1.7.24-jenkins-1-20181114.155326-3</version>
```

Comment:
I recommend adding link to the location of the custom for in comments

Reply:
Added